.env.benefit-backend.example

```diff
@@ -119,3 +119,5 @@ ENABLE_CLAMAV=1
 CLAMAV_URL=http://localhost:8080/api/v1
 AHJO_REQUEST_TIMEOUT=60
 ENABLE_AHJO_AUTOMATION=0
+#For Django 4.2 compatibility
+DJANGO_4_CSRF_TRUSTED_ORIGINS="https://localhost:3000,https://localhost:3100"
```

Comment:
How about a dockerized environment? A docker container cannot access to another with a localhost.

Reply:
The dev environment has nginx running in a container and it proxies both frontends to localhost, I think it is configured to do so in Kesäseteli and TET dev envs as well.